causalpy/experiments.py

```diff
@@ -0,0 +1,38 @@
+#   Copyright 2024 The PyMC Labs Developers
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+"""
+Base class for quasi experimental designs.
+"""
+
+
+class ExperimentalDesign:
+    """Base class for quasi experimental designs."""
+
+    model = None
+    expt_type = None
```

Comment:
why do you need this if they are in the constructor __init__ method?

Reply:
removed in a commit that will soon be pushed